lib/d3.ml

```diff
@@ -181,11 +181,11 @@ module E = struct
   let handle name f = _handler name f
 end
 
-let run selector data t =
+let run ?(node=Js.Unsafe.global##document) t data =
```

Comment:
(If you issue is with the typing, a type cast should do: `(Dom_html.document :> Dom.node)`.